tests/resources/test_project.py

```diff
@@ -350,19 +350,6 @@ def test_list_no_team(session):
     assert 5 == len(projects)
 
 
-def test_list_projects_filters_non_projects(collection, session, caplog):
```

Comment:
The original filtering of payloads which couldn't be deserialized was added to deal with a faulty API (https://github.com/CitrineInformatics/citrine-python/pull/95). Since that's no longer an issue, this test is irrelevant.